src/cnlpt/train_system.py

```diff
@@ -82,6 +108,108 @@ def is_hub_model(model_name: str) -> bool:
     return False
 
 
+def get_dataset_segment(
```

Comment:
move to cnlp_data?

Reply:
Moved to cnlp_data, below the definition of ClinicalNlpDataset.  Maybe it should be a member function, along with the functionality for finding the segment, can look into that and make a PR if that looks like a cleaner option